tooling/api/package.json

```diff
@@ -2,30 +2,71 @@
   "name": "@tauri-apps/api",
   "version": "1.0.0-beta-rc.2",
   "description": "Tauri API definitions",
-  "main": "./index.js",
-  "typings": "./index.d.ts",
+  "type": "module",
   "exports": {
-    ".": "./index.js",
-    "./app": "./app.js",
-    "./cli": "./cli.js",
-    "./dialog": "./dialog.js",
-    "./event": "./event.js",
-    "./updater": "./updater.js",
-    "./fs": "./fs.js",
-    "./path": "./path.js",
-    "./http": "./http.js",
-    "./notification": "./notification.js",
-    "./tauri": "./tauri.js",
-    "./window": "./window.js",
-    "./shell": "./shell.js",
-    "./globalShortcut": "./globalShortcut.js"
+    ".": {
+      "import": "./index.js",
+      "require": "./index.cjs"
+    },
+    "./app": {
+      "import": "./app.js",
+      "require": "./app.cjs"
+    },
+    "./cli": {
+      "import": "./cli.js",
+      "require": "./cli.cjs"
+    },
+    "./dialog": {
+      "import": "./dialog.js",
+      "require": "./dialog.cjs"
+    },
+    "./event": {
+      "import": "./event.js",
+      "require": "./event.cjs"
+    },
+    "./updater": {
+      "import": "./updater.js",
+      "require": "./updater.cjs"
+    },
+    "./fs": {
+      "import": "./fs.js",
+      "require": "./fs.cjs"
+    },
+    "./path": {
+      "import": "./path.js",
+      "require": "./path.cjs"
+    },
+    "./http": {
+      "import": "./http.js",
+      "require": "./http.cjs"
+    },
+    "./notification": {
+      "import": "./notification.js",
+      "require": "./notification.cjs"
+    },
+    "./window": {
+      "import": "./window.js",
+      "require": "./window.cjs"
+    },
+    "./shell": {
+      "import": "./shell.js",
+      "require": "./shell.cjs"
+    },
+    "./globalShortcut": {
+      "import": "./globalShortcut.js",
+      "require": "./globalShortcut.cjs"
+    },
+    "./tauri": {
+      "import": "./tauri.js",
+      "require": "./tauri.cjs"
+    }
   },
   "funding": {
     "type": "opencollective",
     "url": "https://opencollective.com/tauri"
   },
   "scripts": {
-    "build": "rimraf ./dist && rollup -c --silent && copyfiles package.json LICENSE* CHANGELOG.md dist",
+    "build": "rimraf ./dist && rollup -c --silent && copyfiles package.json LICENSE* CHANGELOG.md src/*.ts src/**/*.ts dist",
```

Comment:
```suggestion
    "build": "rimraf ./dist && rollup -c --silent && copyfiles -f package.json LICENSE* CHANGELOG.md src/*.ts dist && copyfiles -f src/helpers/*.ts dist/helpers",
```

Reply:
why?

Reply:
passing the `-f` flag to flatten the `src` directoy into `dist`, otherwise it will create a `src` directory inside `dist`.
and copying the files of `helpers` separately because we don't want them to be in `dist` but in `dist/helpers`

Reply:
this how it is built with your changes
![image](https://user-images.githubusercontent.com/48618675/115764267-a51cee00-a3a5-11eb-8a65-ec4da42d7ead.png)

Reply:
I think having 6 files for each api, one with a different extension (.js, .d.ts, .ts, .mjs) would confuse consumers :joy: if you're crazy enough to use the typescript source directly, just use the src folder

Reply:
if we ship `dist/src` then any bundler will just use `*.d.ts` because it won't find any `*.ts` in the dist directory and it has no idea there is a `src` folder

Reply:
I wonder what kind of bundler config would pick up the typescript files over the .js file